lib/exec.ts

```diff
@@ -114,6 +114,7 @@ export function executeDirect(
             timedOut = true;
             kill();
             streams.stderr += '\nKilled - processing time exceeded';
+            streams.stderr += '\n';
```

Comment:
Please add this to the previous line, likew you did in the `exec-tests.ts`
```suggestion
            streams.stderr += '\nKilled - processing time exceeded\n';
```